src/apps/testapps/testPolygonToCells.c

```diff
@@ -40,19 +40,27 @@ static GeoLoop holeGeoLoop = {.numVerts = 3, .verts = holeVerts};
 static GeoPolygon holeGeoPolygon;
 
 static LatLng emptyVerts[] = {{0.659966917655, -2.1364398519394},
-                              {0.659966917655, -2.1364398519395},
-                              {0.659966917655, -2.1364398519396}};
+                              {0.659966917656, -2.1364398519395},
+                              {0.659966917657, -2.1364398519396}};
```

Comment:
Why did these need to be changed?

Reply:
Possibly they didn't need to - I was trying to adjust some test cases to get better coverage although I don't think I really accomplished that.

Reply:
My mistake, they do need to be changed because as defined this is a straight line. polygonToCells will reject straight lines now - even if a cell centroid is along the line - because it would have attempted to divide by zero when estimating.